x-pack/plugins/enterprise_search/kibana.json

```
@@ -7,5 +7,10 @@
   "optionalPlugins": ["usageCollection", "security", "home", "spaces", "cloud"],
   "server": true,
   "ui": true,
-  "requiredBundles": ["home"]
+  "requiredBundles": ["home"],
+  "owner": {
+    "name": "Enterprise Search",
+    "githubTeam": "enterprise-search-frontend"
+  },
+  "description": "Adds the admin dashboards for the Enterprise Search products"
```

Minor copy suggestion:

```suggestion
  "description": "Adds dashboards for discovering and managing Enterprise Search products"
```

I left out `admin` because that seemed slightly confusing for App Search at least, since we have the concept of non-admin and admin users, and non-admin users can still access our dashboards.

---

I like this better. If Chris want's to change it, I'll open a new PR. Thanks for the review!